public/app/features/templating/pickers/PickerRenderer.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { PureComponent } from 'react';
+import { VariableHide, VariableModel } from '../variable';
+import { e2e } from '@grafana/e2e';
+import { variableAdapters } from '../adapters';
+
+interface Props {
+  variable: VariableModel;
+}
+
+export class PickerRenderer extends PureComponent<Props> {
+  render() {
+    if (!this.props.variable) {
+      return <div>Couldn't load variable</div>;
+    }
```

Comment:
I see pickers implement  the same logic, maybe it can be removed from them

Reply:
Great suggestion! Will look into that

Reply:
fixed!